.circleci/config.yml

```diff
@@ -0,0 +1,64 @@
+version: 2 # use CircleCI 2.0
+jobs: # a collection of steps
+  build: # runs not using Workflows must have a `build` job as entry point
+    parallelism: 4 # run four instances of this job in parallel
+    working_directory: ~/spree # directory where steps will run
+    docker: # run the steps with Docker
+      - image: circleci/ruby:2.4-node-browsers # ...with this image as the primary container; this is where all `steps` will run
+        environment: # environment variables for primary container
+          DB: postgres
+          DB_HOST: localhost
+          DB_USERNAME: postgres
+          CIRCLE_TEST_REPORTS: /tmp/test-results
+          CIRCLE_ARTIFACTS: /tmp/test-artifacts
+          BUNDLE_JOBS: 4
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: ~/spree/vendor/bundle
+      - image: circleci/postgres:9.5-alpine # database image
+        environment: # environment variables for database
+          POSTGRES_USER: postgres
+    steps: # a collection of executable commands
+      - checkout # special step to check out source code to working directory
+
+      # Which version of bundler?
+      - run:
+          name: Which bundler?
+          command: bundle -v
+
+      # Restore bundle cache
+      - restore_cache:
+          keys:
+            - spree-bundle-v2-{{ .Branch }}
```

Comment:
Why `v2` ?

Reply:
@bbonislawski v2 == circle 2.0 :)

Reply:
Let's say it makes sense. Everything else is 👍